var/spack/repos/builtin/packages/py-fenics-ffcx/package.py

```diff
@@ -41,7 +41,7 @@ class PyFenicsFfcx(PythonPackage):
     depends_on("py-fenics-ufl@2022.1.0", type=("build", "run"), when="@0.4.2")
 
     depends_on("py-fenics-basix@main", type=("build", "run"), when="@main")
-    depends_on("py-fenics-basix@0.7", type=("build", "run"), when="@0.7")
+    depends_on("py-fenics-basix@0.7.0:0.7", type=("build", "run"), when="@0.7")
```

Comment:
0.7 and 0.7.0:0.7 are equivalent. I generally prefer the former because it's shorter.

Reply:
Will change.